packages/cli/src/commands/source/push/add.ts

```diff
@@ -62,9 +62,10 @@ export default class SourcePushAdd extends Command {
     const {flags} = this.parse(SourcePushAdd);
     if (!flags.file && !flags.folder) {
       this.error(
-        'You must provide either --file= or --folder=. See source:push:add --help for more information.'
+        'You must minimally set the `file` or the `folder` flag. Use `source:push:add --help` to get more information.'
```

Comment:
Maybe me, but doesn't sounds... "right". Invoking @jpmarceau :P

Reply:
I copied the message from `delete`, which had been revised by Fred Beaudoin.

Reply:
minimally sounds odd, maybe it was just me okok, can be ignored 👍